### PR TITLE
Connect to PSQL database

### PR DESCRIPTION
**Pull request by gcpinckert**

### CODE REVIEW

**Review by westrachel — DISMISSED** [review]
pg connection updates look good

File: db/pg.js [hunk 1]
```diff
@@ -19,3 +19,18 @@
...
+const { Client } = require("pg");
+const client = new Client({
+  user: process.env.PSQL_USERNAME,
+  host: process.env.PSQL_HOST, // update when we open a new TCP tunnel
+  database: process.env.PSQL_DB_NAME,
+  password: process.env.PSQL_PASSWORD,
+  port: process.env.PSQL_PORT, // update when we open a new TCP tunnel
+});
+
+client.connect();
```
Comment (lukeoguro) [inline db/pg.js]: Please update this line with a `console.log`.

**Review by lukeoguro — CHANGES_REQUESTED** [review]
Add `console.log` that we've connected.

**Comment by gcpinckert** [comment]
Added that message you wanted! Take a look

**Review by lukeoguro — APPROVED** [review]
Thanks!

**Review by westrachel — APPROVED** [review]
logging connection progress update looks good

**Pull request merged**